apps/state_mediator/test/state_mediator/integration/gtfs_test.exs

```diff
@@ -262,10 +235,10 @@ defmodule StateMediator.Integration.GtfsTest do
     test "Newburyport/Rockport has 2 non-ignored shapes each direction" do
       [shapes_0, shapes_1] = shapes_in_both_directions("CR-Newburyport")
 
-      assert [%{name: "North Station - Manchester"}, %{name: "North Station - Newburyport"}] =
+      assert [%{name: "North Station - Newburyport"}, %{name: "North Station - Manchester"}] =
```

Comment:
Because of the change in weekend service, the new schedule has more train trips to/from Newburyport than to/from Manchester.